rspace/src/main/scala/coop/rchain/rspace/HotStoreTrieAction.scala

```diff
@@ -18,12 +19,20 @@ final case class TrieInsertConsume[P, K](
     continuations: Seq[WaitingContinuation[P, K]]
 ) extends TrieInsertAction
 
+/** actions flavours with data encoded into ByteVector*/
+final case class TrieInsertBinaryProduce(hash: Blake2b256Hash, data: Seq[ByteVector])
```

Comment:
When we compute merge actions, we have already ByteVector values, so it makes sense to have such TrieActions that do not care about how to serialize rholang data types